Move mod/tagrm.php to src/Module

MrPetovan opened the pull request:
Part of #4090

Also fixes an unreported issue where single tag removal links on a photo edit page wouldn't actually delete the tag.

MrPetovan committed:
Create Post/Tag/Remove module class and route

- Fix tag name parameter in single tag removal link in mod/photos
- Remove "/post" from backend routes as it was preventing sessions from working in this module. No existing module had a route starting with "/post".

MrPetovan committed:
Remove mod/tagrm.php

MrPetovan committed:
Updated main translation file after moving strings

nupplaphil commented on src/Module/Post/Tag/Remove.php:
```diff
@@ -0,0 +1,139 @@
+
+	protected function content(array $request = []): string
+	{
+		$photo_return = $this->session->get('photo_return', '');
```
Wouldn't a `return` parameter inside the GET parameter be more transparent? I know it from other mechanisms, like OIDC where this is working this way

MrPetovan replied:
Ah yes, I got carried away trying to fix this stupid feature nobody uses, and I lost sight of this in the process.

MrPetovan commented:
@nupplaphil Thanks to your change request, I was able to figure out that the module didn't have any session after I moved it from `/photos/tag/remove` to `/post/tag/remove`. It turns out that `/post` was designated a backend route, and it was preventing the regular session from initializing. No existing modules used this route stem so I removed it from the list.
